scipy/stats/_discrete_distns.py

```diff
@@ -142,13 +142,16 @@ class nbinom_gen(rv_discrete):
 
     Notes
     -----
-    The probability mass function for `nbinom` is::
+    Negative binomial distribution describes a sequence of i.i.d. Bernoulli trials, 
+    repeated until a predefined, non-random number of failures occurs. 
```

Comment:
This parametrization, as far as I can see (11:20pm local time so please double-check) corresponds to the series of iid Bernoulli trials repeated until a predefined, non-random number of *successes*, occurs.

This is parametrization 3 in https://en.wikipedia.org/wiki/Negative_binomial_distribution#Alternative_formulations